warehouse/models/mart/transit_database/dim_mobility_mart_providers.sql

```diff
@@ -0,0 +1,163 @@
+{{ config(materialized='table') }}
+
+-- An Organization (i.e. dim_organizations) can provide multiple different
+-- Services (i.e. dim_services). Each Service will have its own funding sources,
+-- GTFS feeds, and target counties.
+
+{% set innerDelimiter = "';'" %}
+
+WITH organizations AS (
+   SELECT * FROM {{ ref('dim_organizations') }}
+   WHERE _is_current
+),
+
+services AS (
+    SELECT * FROM {{ ref('dim_services') }}
+    WHERE _is_current
+),
+
+county_geogs AS (
+    SELECT * FROM {{ ref('dim_county_geography') }}
+    WHERE _is_current
+),
+
+orgs_x_hq AS (
+    SELECT * FROM {{ ref('bridge_organizations_x_headquarters_county_geography') }}
+    WHERE _is_current
+),
+
+services_x_funding AS (
+    SELECT * FROM {{ ref('bridge_services_x_funding_programs') }}
+    WHERE _is_current
+),
+
+gtfs_schedules AS (
+    SELECT
+        key,
+        {{ from_url_safe_base64('base64_url') }} AS base64_url
```

Comment:
[belated] Just noting I found this a bit confusing since at this point this is no longer base64 encoded, might just call it `schedule_url` or similar?